scripts/flatten_collections.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import json
+import os
+
+from linz_logger import get_log
+
+from scripts.cli.cli_helper import format_source
+from scripts.files.files_helper import is_json
+from scripts.files.fs import read, write
+from scripts.stac.imagery.collection import ImageryCollection
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--source", dest="source", nargs="+", required=True)
+    parser.add_argument("--collection_id", dest="collection_id", required=True)
+    parser.add_argument("--destination", dest="destination directory", required=True)
+
+    arguments = parser.parse_args()
+
+    source = format_source(arguments.source)
+
```

Comment:
code style: few blank lines that I don't think are necessary. Blank lines should be used to indicate logical sections.